policy/aggregation_type.go

```diff
@@ -214,6 +265,38 @@ func (aggTypes AggregationTypes) IsValidForTimer() bool {
 	return true
 }
 
+// Quantiles returns all the quantiles found in the list aggregation types.
+func (aggTypes AggregationTypes) Quantiles(p pool.FloatsPool) []float64 {
+	var (
+		res         []float64
+		initialized bool
+		medianAdded bool
+	)
+	for _, aggType := range aggTypes {
+		q, ok := aggType.Quantile()
```

Comment:
Should we return an error in this case instead of swallowing the error silently?